### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by brightlee93**
please review :) @aubiene

### CODE REVIEW

**Comment by brightlee93** [comment]
@awang1993

File: prac_06/programming_language.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class ProgrammingLanguage:
```
Comment (Awang1993) [inline prac_06/programming_language.py:1]: add docstring to outline file

File: prac_06/languages.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from prac_06.programming_language import ProgrammingLanguage
```
Comment (Awang1993) [inline prac_06/languages.py:1]: add docstring to outline file

File: prac_06/guitars.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from prac_06.guitar import Guitar
+
+guitars = []
+counter = 1
```
Comment (Awang1993) [inline prac_06/guitars.py:4]: unused counter

File: prac_06/used_car.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    print("Car {self.fuel}, {self.odometer}".format(self=my_car))
+
+    limo = Car("limo", 100)
+    limo.add_fuel(20)
+    print(limo.fuel)
+    limo.drive(115)
+    print(limo.odometer)
+    print(limo)
+
+
```
Comment (aubiene) [inline prac_06/used_car.py:25]: Code looks good! 
maybe add  print("fuel =", limo.fuel) and print("odo =", limo.odometer),
just makes easier to understand in the output.

**Pull request merged**